### PR TITLE
fix(utils/check) file check for mfusg unstructured models

### LINKED ISSUE #1072: Potential bug when loading USG model
Hey Team,

I tried to use the flopy.modflow.Modflow.load to load my existing robust USG model. It gave me error says " not enough values to unpack (expected 3, got 1)" and screenshot below. I further confirmed the error using USGS test example - folder '01B_nestedgrid_gnc'. Could you please help? Thanks!

Jack Wang

image

### PR DESCRIPTION
**Pull request by cnicol-gwlogic**
mfusg file checking did not previously work
* add get_neighbors function clause for mfusg unstructured grids (in check class, but not mf6check)
* add modflow/mfdisu/_get_neighboring_nodes function
* fix pakbase/_check_flowp hk and vka for unstructured cases
* modify modflow/mfbas/check function to cater for mfusg unstructured grids, in addition to
  structured modflow grids
* modify t016_test.py to cater for modflow/mfdisu/_get_neighboring_nodes function

Resolves #1072, although issues still need to be addressed to enable loading RCH and EVT, so users will need to use "load_only=[]" to avoid trying to load those for now.

### CODE REVIEW

**Comment by cnicol-gwlogic** [comment]
Chris - I rejigged this PR based on your review of #1076. I decided not to move the get_neighbors function to the grid classes because these seem designed to be independent of the model class, and I would need access to the model class / disu for get_neighbors.  I was wary of changing this but let me know if you still prefer it to be moved it out of utils/check.

File: autotest/t016_test.py [hunk 1]
```diff
@@ -46,11 +46,11 @@ def test_usg_disu_load():
     for (key1, value1), (key2, value2) in zip(
         disu2.__dict__.items(), disu.__dict__.items()
     ):
-        if isinstance(value1, flopy.utils.Util2d) or isinstance(
-            value1, flopy.utils.Util3d
-        ):
+        if isinstance(value1, (flopy.utils.Util2d, flopy.utils.Util3d):
```
Comment (mwtoews) [inline autotest/t016_test.py:49]:
```suggestion
        if isinstance(value1, (flopy.utils.Util2d, flopy.utils.Util3d)):
```

**Comment by langevin-usgs** [comment]
Hey @cnicol-gwlogic, I took a quick look through this, and it looks like you've made some nice improvements.  Is there one or more simple tests that show that this is all working as intended?  Or do you think the t016_test is sufficient?

**Comment by cnicol-gwlogic** [comment]
Hi @langevin-usgs, yep sorry - should have thought of that, given that's how I was testing the changes at my end. doh.See last commit. I've got a couple more tests ready for another PR that will add RCH and EVT loading capability. Cheers.

**Pull request merged**